modules/distr-tracing-tempo-configuring-tempostack-metrics-and-alerts.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * distr-tracing-tempo-configuring.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-tempostack-metrics-and-alerts_{context}"]
+= Configuring TempoStack metrics and alerts
+
+You can enable metrics and alerts of TempoStack instances.
+
+.Prerequisites
+
+* Monitoring for user-defined projects is enabled in the cluster. See xref:../../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects].
+
+.Procedure
+
+. To enable metrics of a TempoStack instance, set the `spec.observability.metrics.createServiceMonitors` field to `true`:
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: <name>
+spec:
+  observability:
+    metrics:
+      createServiceMonitors: true
+----
+
+. To enable alerts for a TempoStack instance, set the `spec.observability.metrics.createPrometheusRules` field to `true`:
++
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: <name>
+spec:
+  observability:
+    metrics:
+      createPrometheusRules: true
+----
+
+.Verification
+
+You can use the *Administrator* view of the OpenShift web console to verify successful configuration:
```

Comment:
```suggestion
You can use the *Administrator* view of the {product-title} web console to verify successful configuration:
```
Related: https://github.com/openshift/openshift-docs/pull/62410#discussion_r1318989879

Reply:
Thank you. Fixed.